### ISSUE TITLE
  A little question about pre-training

### ISSUE THREAD

**Issue by yyc776** [SukwonYun/LTE4G#1]
Hello,Thank you so much for sharing！

The code at models/lte4g.py [line 98](https://github.com/SukwonYun/LTE4G/blob/main/models/lte4g.py#L98) uses all the data, including tests and validation .
 
I wonder if pre-training with data from test sets and validation modules will have an impact .
If there is no impact, is it because the loss function is constructed by the adjacency matrix, not by the label ?

Thank you very much!

**Comment by SukwonYun** [comment]
Hi, @yyc776.
Thank you for your interest in our work.

As you mentioned, the loss function is constructed by the mean squared loss between the original adjacency matrix and its predicted matrix. Since we only used feature information and its adjacency, it is fine. (i.e., you can think of it as in the perspective of a semi-supervised setting.) For more details, you can refer to its original source below.

[WSDM '21] GraphSMOTE: Imbalanced Node Classification on Graphs with Graph Neural Networks
 [https://arxiv.org/abs/2103.08826](https://arxiv.org/abs/2103.08826)

**Comment by yyc776** [comment]
Your answer is very useful. Thank you very much for answering my question.
Have a good day！
